specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -2002,6 +2002,9 @@
     },
     "DiskEncryptionSetUpdateProperties": {
       "properties": {
+        "encryptionType": {
```

Comment:
just to confirm, `encryptionType` isn't a required parameters, otherwise it's required to be added in new API versoin since it will cause existing users breaking.

Reply:
offline double confirmed, it's optional property